sdk/storage/Azure.ResourceManager.Storage/tests/Tests/StorageAccountTests.cs

```diff
@@ -69,8 +69,8 @@ public async Task CreateDeleteStorageAccount()
         public async Task GetAllStorageAccounts()
         {
             //create two storage accounts
-            string accountName1 = Recording.GenerateAssetName("storage1");
-            string accountName2 = Recording.GenerateAssetName("storage2");
+            string accountName1 = await CreateValidAccountNameAsync("teststoragemgmt1");
```

Comment:
since the Recording.GenerateAssetName will generate a random name, I don't think here need have '1‘ and '2' prefix